facefusion/face_selector.py

```diff
@@ -21,6 +21,7 @@ def find_similar_faces(faces : List[Face], reference_faces : FaceSet, face_dista
 
 def compare_faces(face : Face, reference_face : Face, face_distance : float) -> bool:
 	current_face_distance = calc_face_distance(face, reference_face)
+	current_face_distance = float(numpy.interp(current_face_distance, [ 0, 2 ], [ 0, 1 ]))
```

Comment:
I think numpy.interp() already returns float

Reply:
no it returns ndarray